src/main/java/uk/ac/ebi/eva/contigalias/controller/contigalias/ContigAliasController.java

```diff
@@ -291,10 +293,10 @@ public static void linkPagedModelGetSequencesByAssemblyAccession(
             @RequestParam(required = false) @ApiParam(value = "Taxonomic ID of a group of accessions. Eg: 9606") Optional<Long> taxid,
             @RequestParam(required = false, name = "accession") @ApiParam(value = "Genbank or Refseq assembly " +
                     "accession. Eg: GCA_000001405.10") Optional<String> asmAccession,
-            @RequestParam(required = false, name = "name") @ApiParam(value = "Specify if the provided name is a " +
-                    "chromosome name or a UCSC style name. The acceptable param values are " + NAME_SEQUENCE_TYPE +
-                    " " +
-                    "and " + NAME_UCSC_TYPE + " respectively. If this parameter is omitted then the name is assumed " +
+            @RequestParam(required = false, name = "name") @ApiParam(value = "Specify if the provided name is an " +
+                    "NCBI chromosome name, ENA name, or a UCSC style name. The acceptable param values are " +
```

Comment:
Judgment call: I am not sure if it should be called NCBI chromosome name unless we are referring specifically to a RefSeq chromosome naming convention (see https://bioinformatics.stackexchange.com/q/27).

Reply:
I think it's at least [sometimes](https://www.biostars.org/p/51643/#425222) referred to as such... but I could name it something else if it would be more appropriate.  Naming things is hard (which I guess is the entire point of this service).

Reply:
Yes, I agree it is hard. The reason why we are calling it `NCBI chromosome name` is because that is the chromosome name that comes in the assembly report provided by NCBI. Just as we are naming `ENA name` because of the same reason. We can discuss the naming to agree on that

Reply:
Renamed to `genbank` throughout as per this morning's discussion.